apps/src/applab/applab.js

```diff
@@ -302,7 +306,10 @@ function queueOnTick() {
 
 function onDebugInputKeyDown(e) {
   if (e.keyCode == KeyCodes.ENTER) {
+    e.preventDefault();
```

Comment:
as far as I can remember the enter event creates a `<br>` or a new `<div>`, and the `e.textContent` method that is currently used to update/erase doesn't handle the line breaks well.